src/Livewire/Forms/Contact.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace FluxErp\Livewire\Forms;
+
+use FluxErp\Actions\Address\CreateAddress;
+use FluxErp\Actions\Address\UpdateAddress;
+use FluxErp\Actions\Contact\CreateContact;
+use FluxErp\Actions\Contact\UpdateContact;
+use FluxErp\Models\Client;
+use FluxErp\Models\Language;
+use Illuminate\Support\Arr;
+use Livewire\Form;
+
+class Contact extends Form
```

Comment:
property order seems a little bit random

Reply:
now ordered by contact and address table (the form holds both basically)